expo-maps/ios/ExpoMaps/ExpoAppleMapsModule.swift

```diff
@@ -9,24 +9,34 @@ public class ExpoAppleMapsModule: Module {
       view {
         AppleMapsView()
       }
+
```

Comment:
Cosmetics; inconsistent formatting.
These are spaced out here, but not in `ExpoGoogleMapsModule.swift`.